### PR TITLE
add basic set up + classes for utility framework

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Phase 1 of the utility framework - the basic set up and classes for the utility framework. The framework uses functional CSS which favors small, single-purpose classes with names based on visual function.

### CODE REVIEW

File: src/utilities/visibility.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  visibility: visible;
+}
+
+.visibility-sr-only {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin: 0;
+  overflow: hidden;
+  visibility: hidden;
```
Comment (ethanWallace) [inline src/utilities/visibility.scss:17]: It seems  `visibilty: hidden` also removes the element from the accessibility tree so it won't be picked up by assistive technology.

File: src/utilities/spacing.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// --- Margin + Padding Spacing Classes ---
+
+@each $spaceClass, $space in $spacing {
+  // classes for all sides
+  .m-#{$spaceClass} {
+    margin: $space;
+  }
+
+  .p-#{$spaceClass} {
+    margin: $space;
```
Comment (ethanWallace) [inline src/utilities/spacing.scss:12]: `.p-{size}` classes are setting margin instead of padding

**Review by ethanWallace — CHANGES_REQUESTED** [review]
This looks great! Just noticed a couple things that need to be updated

File: src/utilities/visibility.scss (hunk 1 above)
Comment (melaniebmn) [reply src/utilities/visibility.scss]: @ethanWallace updated!

File: src/utilities/spacing.scss (hunk 2 above)
Comment (melaniebmn) [reply src/utilities/spacing.scss]: @ethanWallace good catch, updated!

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Pull request merged**